src/metabase/models/card.clj

```diff
@@ -547,16 +547,22 @@
     (parameter-card/upsert-or-delete-from-parameters! "card" (:id card) (:parameters card))))
 
 (t2/define-before-update :model/Card
-  [card]
+  [{:keys [verified-result-metadata?] :as card}]
   ;; remove all the unchanged keys from the map, except for `:id`, so the functions below can do the right thing since
   ;; they were written pre-Toucan 2 and don't know about [[t2/changes]]...
   ;;
   ;; We have to convert this to a plain map rather than a Toucan 2 instance at this point to work around upstream bug
   ;; https://github.com/camsaul/toucan2/issues/145 .
-  (-> (into {:id (:id card)} (t2/changes card))
+  (-> (into {:id (:id card)} (t2/changes (dissoc card :verified-result-metadata?)))
```

Comment:
Having to add and then remove this data seems a bit noisy. How about `update-meta` on the card instead of adding a new key to the card itself?

Reply:
I'm a bit vary of relying on `meta` being preserved. Just naively putting metadata at the point where I now `assoc` `:verified-result-metadata?` is not enough, that metadata doesn't reach the before update hook (i.e., `card` doesn't have it).